.NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs

```diff
@@ -1,98 +1,98 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-
-namespace Microsoft.Recognizers.Text.Matcher
-{
-    public class AcAutomaton<T> : AbstractMatcher<T>
-    {
-        protected readonly AaNode<T> root = new AaNode<T>();
-
-        public AcAutomaton()
-        {
+﻿using System.Collections.Generic;

```

Comment:
Please, check **AcAutomaton** and **TrieTree**. This appears to be a line ending change.

Reply:
Done!